### PR TITLE
Set correct default config file path

### LINKED ISSUE #79: Error: default config file not found
When cytominer-database is called without explicit config file 
```
cytominer-database ingest images test.sqlite
```
the error 
```
Error: Invalid value for "-c" / "--config-file": Path "/usr/local/lib/python3.6/site-packages/cytominer_database/commands/config/config_htqc.ini" does not exist.
```
is thrown. 

Installed cytominer-database
```
pip install cytominer-database==0.0.1b4
```

### PR DESCRIPTION
**Pull request by mcquin**
Resolves #79

### CODE REVIEW

File: tests/commands/test_command_ingest.py [hunk 1]
```diff
@@ -68,4 +69,54 @@ def test_run(dataset, runner):
...
+
+        result = runner.invoke(cytominer_database.command.command, opts)
+
+        assert result.exit_code == 0
+
+        config = configparser.ConfigParser()
+
+        config_file = pkg_resources.resource_filename(
+            "cytominer_database",
+            os.path.join("config", "config_htqc.ini")
```
Comment (shntnu) [inline tests/commands/test_command_ingest.py]: This config file works only for the `htqc` fixture
For `cellpainting`, it is `config_cellpainting.ini`

I just noticed that all the tests are being performed only on the `htqc`, not `cellpainting`. 

We'd need to look up how it was configured previously and have it test for both dataset fixtures. This test will currently fail on `cellpainting`

**Review by shntnu — CHANGES_REQUESTED** [review]
tests should run on both, `htqc` and `cellpainting`. Currently, they are running only on `htqc`

File: tests/commands/test_command_ingest.py (hunk 1 above)
Comment (mcquin) [reply tests/commands/test_command_ingest.py]: @shntnu I've submitted #81 which runs the tests on both the `htqc` and `cellpainting` datasets.

Is this configuration file the correct default configuration file for the command line? If not, which configuration file should be used? Or, is the `--config` flag meant to work like the `--dataset` flag?

Comment (shntnu) [reply tests/commands/test_command_ingest.py]: Ah! I didn't realize you are testing defaults here. Apparently `test_run_defaults` was not obvious enough for me 😆 
The default config file should be `config_cellpainting.ini`. That means this test should be tested only on `cellpainting` and not `htqc`. But I don't know how you'd configure that 
@mcquin

Comment (shntnu) [reply tests/commands/test_command_ingest.py]: My guess is you'd have `@pytest.mark.skipif(...)` before `def test_run_defaults`, where `...` is a check for `dataset == htqc`

Comment (mcquin) [reply tests/commands/test_command_ingest.py]: Unfortunately, `dataset` isn't available in the context of `skipif` (it'd be really nice if it were). I've split the datasets out into their own fixtures for finer control.

File: tests/commands/test_command_ingest.py [hunk 2]
```diff
@@ -72,15 +72,15 @@ def test_run(dataset, runner):
                        dataset["ingest"]["{}_nrows".format(table_name)]
 
 
-def test_run_defaults(dataset, runner):
+def test_run_defaults(cellpainting, runner):
```
Comment (shntnu) [inline tests/commands/test_command_ingest.py]: Great idea!

**Review by shntnu — APPROVED** [review]
Awesome fix! This is actually easier to read that the skipif option that could have been

**Pull request merged**